research/src/sources/bootstrap.json5

```diff
@@ -421,6 +421,41 @@
           "image": "bootstrap-tooltip-html.png"
         }
       ]
+    },
+    {
+      "name": "Img",
+      "openUIName": "Image",
+      "url": "https://getbootstrap.com/docs/4.0/content/images/",
+      "concepts": [
+        {
+          "name": "class-img-fluid",
+          "openUIName": "fit",
```

Comment:
Although the HTML class name is `img-fluid` we want to enter the concept name as just `fluid`. This is because the `img-` prefix is not relevant to the concept being applied to image, but is just a prefix for this component name. The concept being applied to the image is `fluid`.

This will apply to all concepts named `class-*`.

Reply:
Makes sense, thanks!